### PR TITLE
allow elements missing closing tags (#38)

### PR DESCRIPTION
**Pull request by dinhlongviolin1**
Fix issue #38 and:
- Menu still persisted even if md has been removed (#41)
- better md parsing (only parse markdown content rather than the whole html body) (#40)

### CODE REVIEW

**Review by FabienLelaquais — APPROVED** [review]
🤔if this works... it works.

File: markdownPreview/src/index.tsx [hunk 1]
```diff
@@ -15,14 +15,19 @@ import { createRoot } from "react-dom/client";
...
 function init() {
-    const rootDiv = document.createElement("div", {});
-    rootDiv.setAttribute("id", "root");
-    const bodyInnerHTML = document.body.innerHTML;
-    const baseHref = (document.querySelector('base') || {}).href || "";
-    document.body.innerHTML = "";
-    document.body.appendChild(rootDiv);
+    const mdbd = document.getElementsByClassName("markdown-body");
+    const markdownBodyHTML = mdbd[mdbd.length - 1].outerHTML;
+    mdbd[mdbd.length - 1].remove();
```
Comment (FredLL-Avaiga) [inline markdownPreview/src/index.tsx:20]: Waow ! Heavy stuff

**Review by FredLL-Avaiga — APPROVED** [review]
😶‍🌫️

File: markdownPreview/src/index.tsx (hunk 1 above)
Comment (dinhlongviolin1) [reply markdownPreview/src/index.tsx:20]: need to remove it so we have a clean html body :D

**Pull request merged**